.github/workflows/test.yml

```diff
@@ -63,20 +63,6 @@ jobs:
       run: |
         go build -v .
 
-  generate:
```

Comment:
This hasn't been removed, it has moved to the "code-check.yml" file with the other linters